File: rivu jackson/test1.java
class test1{
    public void check(){
        /*
        int a=10;
        switch(a){
        case 10:System.out.print(a+++" ");
        case 11:System.out.print(++a+" ");
        case 12:System.out.print(a+++" ");
        break;
        default: System.out.println(a++);
        }
        System.out.println(a);

        int a=10,b=20;
        if((a<b)||(a=5)>10)
        System.out.print(b);
        else
        System.out.print(a);
         */
        //System.out.print(-2+10%4+12);
        /*
        int k=5,j=9;
        k+=k++-++j+k;
        System.out.println(k);
        System.out.println(j);

        for(int m=5;m<=20;m+=5){
        if(m%3==0)
        break;
        else
        if(m%5==0)
        System.out.print(m+" ");
        continue;
        }
        char c='a';
        switch(c){
        case 'a':System.out.print("A");
        case 'b':System.out.print("B");
        default:System.out.print("C");
        }
         */
        if(5>10||7==7)
            System.out.println("t");
        else
            System.out.println("f");
    }

    public static void main(){
        test1 o=new test1();
        o.check();
    }
}

File: Problem Statements/Interview Questions/CodeVita/MockVita1/test1.cpp
#include<iostream>
using namespace std;
void Combi(char a[], int reqLen, int s, int currLen, bool check[], int l)
// print the all possible combination of given array set
{
   if(currLen > reqLen)
   return;
   else if (currLen == reqLen) {
      cout<<"\t";
      for (int i = 0; i < l; i++) {
         if (check[i] == true) {
            cout<<a[i]<<" ";
         }
      }
      cout<<"\n";
      return;
   }
   if (s == l) {
      return;
   }
   check[s] = true;
   Combi(a, reqLen, s + 1, currLen + 1, check, l);
      //recursively call Combi() with incremented value of ‘currLen’ and ‘s’.
   check[s] = false;
   Combi(a, reqLen, s + 1, currLen, check, l);
      // recursively call Combi() with only incremented value of ‘s’.
}
int main() {
   int i,n;
   bool check[n];
   cout<<"Enter the number of element array have: ";
   cin>>n;
   char a[n];
   cout<<"\n";
   for(i = 0; i < n; i++) {
      cout<<"Enter "<<i+1<<" element: ";
      cin>>a[i];
      check[i] = false;
   }
   for(i = 1; i <= n; i++) {
      cout<<"\nThe all possible combination of length "<<i<<" for the given array set:\n";
      Combi(a, i, 0, 0, check, n);
   }
   return 0;
}